Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -212,6 +212,10 @@ public function processRequest(ActionRequest $request, ActionResponse $response)
         }
 
         $this->callActionMethod();
+
+        if ($this->response->getContentType() === '') {
```

Comment:
This should cater for the `null` case I suppose:
```suggestion
        if (empty($this->response->getContentType()) {
```
even nicer would be
```php
if (!$this->response->hasContentType()) {
```
so that we can move the empty-logic into the response

Reply:
`empty()` will not suffice, because if `$this->contentType` is `null`, calling the method will throw a fatal TypeError, as it expects to return a `string` (and even though we don't have strict_types on this file yet, that doesn't help for return types).